features/PSE001_Ifcpropertyset-validation.feature

```diff
@@ -0,0 +1,39 @@
+@implementer-agreement
+@PSE
+Feature: PSE001 - IfcPropertySet validation
+The rule verifies that each IfcPropertySet starting with Pset_ is defined correctly.
+
+    Scenario: Agreement on each IfcPropertySet correctly defining an applicable entity.
+
+      Given A file with Schema Identifier "IFC2X3"
+      And An IfcPropertySet
+      And Its attribute Name starts with Pset_
+      Then The IfcPropertySet Name attribute value must use predefined values according to the IFC2x3_definitions.csv table
+      Then The IfcPropertySet must be assigned according to the property set definitions table IFC2x3_definitions.csv
+      Then Each associated IfcProperty must be named according to the property set definitions table IFC2x3_definitions.csv
+      Then Each associated IfcProperty must be of type according to the property set definitions table IFC2x3_definitions.csv
+      Then Each associated IfcProperty value must be of data type according to the property set definitions table IFC2x3_definitions.csv
+
+
+    Scenario: Agreement on each IfcPropertySet correctly defining an applicable entity.
+
+      Given A file with Schema Identifier "IFC4"
+      And An IfcPropertySet
+      And Its attribute Name starts with Pset_
+      Then The IfcPropertySet Name attribute value must use predefined values according to the IFC4_definitions.csv table
+      Then The IfcPropertySet must be assigned according to the property set definitions table IFC4_definitions.csv
+      Then Each associated IfcProperty must be named according to the property set definitions table IFC4_definitions.csv
+      Then Each associated IfcProperty must be of type according to the property set definitions table IFC4_definitions.csv
+      Then Each associated IfcProperty value must be of data type according to the property set definitions table IFC4_definitions.csv
+
+
+    Scenario: Agreement on each IfcPropertySet correctly defining an applicable entity.
+
+      Given A file with Schema Identifier "IFC4X3"
```

Comment:
I think we should be careful here with ifc 4 versioning, but that's a topic for at the other PR perhaps 
https://github.com/buildingSMART/ifc-gherkin-rules/pull/89

Reply:
That's a good point. I'm changing `IFC4X3` to `IFC4X3_ADD2`. Thank you.